inc/model.class.php

```diff
@@ -1497,10 +1504,59 @@ static function showLogResults($models_id) {
                echo "<td>".nl2br($result['status_message'])."</td>";
                echo "<td>".$result['type']."</td>";
                echo "<td>".$result['url']."</td><tr>\n";
+               if(!isset($resume[$result['status']][$result['type']])) {
+                  $resume[$result['status']][$result['type']] = 0;
+               }
+               $resume[$result['status']][$result['type']]++;
+               $nblines++;
             }
             echo "</table></div>\n";
             echo "<script type='text/javascript'>document.getElementById('log1_table').style.display='none'</script>";
          }
+
+         if (!empty($resume)) {
+            echo "<table>\n";
+            echo "<tr>";
+            echo "<td style='width:30px'>";
+            echo "<a href=\"javascript:show_log('2')\"><img src='../pics/minus.png' alt='minus' id='log2'>";
+            echo "</a></td>";
+            echo "<td style='width: 450px;font-size: 14px;font-weight: bold;padding-left: 20px'>".
+               __('Global report', 'datainjection')."</td>";
+            echo "<td style='width: 450px;font-style:italic'>" . __('Number of lines processed', 'datainjection') . " : " . $nblines . "</td>";
+            echo "</tr>\n";
+            echo "</table>\n";
+
+            echo "<div id='log2_table'>";
+            echo "<table class='tab_cadre_fixe center'>\n";
+            echo "<th></th>"; //Icone
+            echo "<th>" . __('Data Import', 'datainjection') . "</th>"; //Import des données
```

Comment:
Comment seems useless here.
```suggestion
            echo "<th>" . __('Data Import', 'datainjection') . "</th>";
```